h2o-automl/src/test/java/hex/ModelMojoWriterTest.java

```diff
@@ -0,0 +1,67 @@
+package hex;
+
+import ai.h2o.automl.targetencoding.BlendingParams;
+import ai.h2o.automl.targetencoding.TargetEncoder;
+import org.junit.Assert;
+import org.junit.BeforeClass;
+import org.junit.Test;
+import water.Key;
+import water.TestUtil;
+import water.fvec.Frame;
+
+import java.io.File;
+import java.io.FileOutputStream;
+import java.util.Map;
+
+/**
+ * This test should be moved to h2o-core module once we move all TargetEncoding related classes there as well. 
+ */
+public class ModelMojoWriterTest extends TestUtil implements ModelTestCommons {
+
+  @BeforeClass public static void stall() { stall_till_cloudsize(1); }
+  Frame trainFrame = parse_test_file("./smalldata/gbm_test/titanic.csv");
```

Comment:
yes, this is not a good idea, this will not pass the keys leak check I think, move this to @Before

Reply:
@seb-h2o If I wanted to share this file between tests first I would have more then one test in this class :) And anyway even with @Before @After you still need global variable so that tests can access it.  So I was planning to add more tests and reuse the frame( of course keeping in mind that I should not mutate the frame) and it is I believe is fine because as far as I know we run tests within test class sequentially. 
@honzasterba It would not pass keys leakage test only because I had TODO to do clean up later. As far as I understand it has nothing to do with whether I'm using before or not.
But I agree that given the fact that I have only one test I should have put parsing frame logic just inside the test's method to not confuse anyone.

Reply:
@seb-h2o you can see that I usually do clean up in @afterEach as in `MojoTESupportIntegrationTest` for example. This helps me not to do it manually in each test and we quite often have at least one frame to work with in the tests.